src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
```

Comment:
you can select whole tree and operate later on his children:

`const tree = document.querySelector('.tree');`